sdk/eventhub/eventhubs-checkpointstore-blob/package.json

```diff
@@ -116,9 +117,8 @@
     "rollup-plugin-sourcemaps": "^0.4.2",
     "rollup-plugin-terser": "^5.1.1",
     "rollup-plugin-visualizer": "^2.0.0",
-    "typescript": "^3.2.2",
     "ts-node": "^8.3.0",
-    "util": "^0.12.1",
-    "guid-typescript": "1.0.9"
```

Comment:
>"guid-typescript": "1.0.9"

caret `^` for this, @chradek?

Reply:
Yeah, anything we depend on should have a caret dependency according to @mikeharder.

Reply:
Correct, our guideline is to use caret unless there's a specific reason not to.